web/react/components/delete_post_modal.jsx

```diff
@@ -149,10 +157,10 @@ export default class DeletePostModal extends React.Component {
                         {'Cancel'}
                     </button>
                     <button
+                        ref='deletePostBtn'
                         type='button'
                         className='btn btn-danger'
                         onClick={this.handleDelete}
-                        autoFocus='autofocus'
```

Comment:
Yeah setting `autoFocus` to true doesn't work sadly (although I thought it wasn't working for a different reason than @hmhealey mentioned, that's good to know thanks!).  And yeah I wanted to keep opening the delete box consistent with how it's normally opened w/ the dispatcher rather than have two separate routes and this worked.